### DISCUSSION TITLE
COM port not working in cygwin

### DISCUSSION THREAD

**Discussion by Pitichinaccio** [stronnag/mwptools#95] [General]
Hello Jonathan,

when I open mwp the list of available devices is empty. 

I can connect to my FC via the INAV configurator and it shows that it's on COM9 @57600 baud.

So I just typed "/dev/ttyS8@57600" into the device field and click connect, but nothing happens.
I can't upload a mission, so it seems that there still is no connection. 

If I type e.g. "/dev/ttyS7" or any other number there comes an error message. 

Bernd

**Comment by stronnag** [comment]
1. The list of devices will always be empty; cygwin does not support the Linux device enumeration.
2. On Win10, `/dev/ttyS2@57600` works (in a VM with USB passthrough). OTOH, it's a VCP FC so baud rate is somewhat meaning less.
3.  I have no idea whether or why this would not work on Wn7.

**Comment by stronnag** [comment]
Looking back through previous revisions of the wiki, at one stage there was not need to modify the device number (COMx == /dev/ttySx). Whether this was due to an older Windows (Win7) or older cygwin, I have no idea.

**Comment by Pitichinaccio** [comment]
Thanks for the hint. Device renumbering seems ok. It's on COM9 but when I use ttyS9 I get an error message.
